src/main/java/com/streamr/client/dataunion/JavaWrapperUtil.java

```diff
@@ -79,7 +79,10 @@ public static void main(String[] args){
                         public boolean accept(File file) {
                             String fname = file.getName();
                             return fname.endsWith(".json")  &&
-                                    (fname.startsWith("DataUnion") || fname.contains("IERC20") || fname.contains("AMB")) ;
+                                    (fname.startsWith("DataUnion") ||
+                                            fname.contains("IERC20") ||
+                                            fname.startsWith("Binance") ||
+                                        fname.contains("AMB")) ;
```

Comment:
maybe add a comment what classes are supposed to be matched here; maybe instead of string matching with `startsWith` and `contains` just match against an explicit list of strings